lib/ansible/constants.py

```diff
@@ -100,7 +100,7 @@ def __getitem__(self, y):
 BOOL_TRUE = BOOLEANS_TRUE
 COLLECTION_PTYPE_COMPAT = {'module': 'modules'}
 
-PYTHON_DOC_EXTENSIONS = ('.py', '.pyc', '.pyo')
+PYTHON_DOC_EXTENSIONS = ('.py',)
```

Comment:
any reason we want to remove compiled versions?

Reply:
`read_docstub()` was trying to read .pyc files which caused warnings like `[WARNING]: ansible.builtin.urlsplit has a documentation formatting error: 'utf-8' codec can't decode byte 0xf3 in position 1: invalid continuation byte`. @felixfontein caught the bug by having a `ansible/lib/ansible/plugins/test/core.pyc` file and it prevented listing any builtin filters/tests.

Reply:
i wonder if that ever worked

Reply:
I don't *think* so. But it only became an issue once we started loading adjacent documentation.